### ISSUE TITLE
Clustered influxdb

### ISSUE THREAD

**Issue by jchauncey** [deis/influxdb#1]
It would be nice if our influxdb image could scale up/down as needed using replication controllers.

**Comment by arschles** [comment]
Any idea what's gonna be necessary to do so? For example, there's gonna probably be a bootstrap problem to solve, and also gonna have to determine influx pod membership to join new nodes.

**Comment by jchauncey** [comment]
Nope. This will probably be fairly hard and I think we should reach out to
the influx guys to help us.
